core/src/main/scala/cats/data/NonEmptyMapImpl.scala

```diff
@@ -317,7 +322,7 @@ private[data] sealed abstract class NonEmptyMapInstances {
   implicit def catsDataShowForNonEmptyMap[K: Show, A: Show]: Show[NonEmptyMap[K, A]] =
     Show.show[NonEmptyMap[K, A]](_.show)
 
-  implicit def catsDataBandForNonEmptyMap[K, A]: Band[NonEmptyMap[K, A]] = new Band[NonEmptyMap[K, A]] {
+  implicit def catsDataSemilatticeForNonEmptyMap[K, A]: Semilattice[NonEmptyMap[K, A]] = new Semilattice[NonEmptyMap[K, A]] {
```

Comment:
you changed it to semilattice here but not the law testing? I am not sure if `Map` combine is commutative.

Reply:
Oh, right, yeah that was dumb, I'll check that one later :)